lib/musl/time/mktime.c

```diff
@@ -24,7 +24,7 @@ time_t mktime(struct tm *tm)
 	*/
 	return t;
 
-error:
+//error:
```

Comment:
why remote the goto label? is it not required (no I didn't look at the source :( )

Reply:
Compiler said it wasn't used.
Thought to just comment it out in case someone looks for it in the future again.